desktop_version/src/Graphics.cpp

```diff
@@ -3026,6 +3045,7 @@ void Graphics::reloadresources(bool fast /*= false*/) {
     Makebfont();
     pre_fakepercent.store(95);
     makecustomtilearray();
+	makecustomspritearray();
```

Comment:
This is indented with a tab when the surrounding code is indented with 4 spaces.